lib/internal/Magento/Framework/Image/Adapter/Gd2.php

```diff
@@ -63,7 +61,7 @@ protected function _reset()
      */
     public function open($filename)
     {
-        if (!file_exists($filename)) {
+        if (!$filename || !file_exists($filename)) {
```

Comment:
Hello @anzin 
Please use strict condition for null

```suggestion
        if ($filename === null || !file_exists($filename)) {
```

Reply:
Code fixed by recommendation, please check if every think is well.

Thank you.